portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsUserCollaboration.testcase

```diff
@@ -123,6 +123,54 @@ definition {
 		PublicationsPermissions.assertEditorPermissions(publicationName = "Publication Name 2");
 	}
 
+	@description = "This is a use case for LPS-195715. Can view Publication with link as Guest user."
+	test CanReviewPublicationWithLinkAsGuest {
```

Comment:
Can you add the feature flag property to all of the tests you have written? If not the feature won't appear.